slajkoa_hf_v2.html

```diff
@@ -0,0 +1,110 @@
+    <p>Szintén a szomszédban van a hernyóra, esetleg sokszemű csigára emlékeztető grazi Kunsthaus. A Peter Cook és Colin Fournier által tervezett amorf épület Fournier „barátságos földönkívülinek” nevezte. A Kunsthaus a 2003-as Európa Kulturális Fővárosa program részeként épült, mára beágyazódott a város szövetébe.</p>
+    <p>A magyaroknak talán a legismertebb példa a prágai Dancing House Hotel, amelyet Frank Gehry tervezett, és fennállása óta az egyik legfontosabb prágai fotómotívummá nőtte ki magát. Van olyan épület is, amelyen csak a levegőből lehet észlelni a hasonlóságot: ennek ékes példája Reykjavíkban a Smáralind vagy ahogy angolul elterjedt, a Penis Mall, amely föntről egy péniszre emlékeztet.
+        Kritikusai szerint a mimetikus épületeket nehéz komolyan venni, és elég giccsesek.
+        Van azonban néhány utánzás, vagy ha úgy tetszik, mimézis, amely finomabb, visszafogott motívumokkal utal az épület funkciójára</p>
+    <br>
+    <img src="https://assets.telex.hu/images/20240321/1711019359-temp-iGPjOn_cikktorzs:xl@1.25x.jpg" height="200px" width="300px" alt="">
+    <img src="https://assets.telex.hu/images/20240321/1711019368-temp-LFNhiH_cimlap-full@4x.jpg" height="200px" width="300px" alt="">
+
+    <p>Ott van például a tokiói <span class="kiemel">Asahi</span> sörgyár központja. A Nikken Seikei által tervezett és 1989-ben elkészült, 22 emeletes Asahi Beer Azumabashi épületet sörformájúra tervezték. A borostyánsárga üveg a sört idézi, míg a tetején lévő fehér panelek a habot. A szomszédban található az Asahi Super Dry Hall. A francia sztárépítész, Philippe Starck által tervezett, 1990-ben épült, zömök, fekete épület tetejét egy 300 tonnás aranyszínű láng díszíti, amely egyesek szerint úgy néz ki, mint a frissen csapolt sör masszív habja. Tokióban más asszociációk is felütötték a fejüket: az épületet aranykakinak vagy aranyszarnak is nevezik. A források szerint szeretetteljesen.</p>
+    <p>Nálunk nem megalomán épületekben ölt testet, de akad néhány apróbb példa: ilyen Makovecz Imre Cápa néven elhíresült kisvendéglője Velencén, a szentesi Cicaház, ahol macskákat árulnak egy virágbolt üzemel vagy a repülőtér 1-es terminálja, amely felülnézetből repülőgépet formáz. Hasonlóan a levegőből nyeri el jelentését a Budaörsi út 75. alatti Yacht irodaház.
+        A soha el nem készülő, a Rákóczi híd budai hídfőjéhez tervezett, Rubik-kocka alakú kiállítási központ viszont épp monumentális alkotás lenne/lett volna/lesz majd valamikor.</p></div>
+    
+   <br>     
+   Cikk küldése ismerősnek:
+
+   <form action="#" method="get">
+    <div>
+        <label for="név">Név*</label>
+        <input type="text" id="név" placeholder="Név" required>
+    </div>
+    <div>
+        <label for="telefonszám">Telefonszám</label>
+        <input type="tel" id="telefonszám" >
+    </div>
+    <div>
+        <label for="email">Email*</label>

```

Comment:
A label `for` attribútuma és a hozzá tartozó input `id` attribútuma ugyanazon nevet kellene felvegyék a működéshez. Itt, és a következőnél különbözik, így nem tud fókuszt adni.